server/db.js

```diff
@@ -215,6 +248,38 @@ exports.createBackend = function createBackend(config) {
         });
       });
     },
+
+    getOps(collabid, filepath, cutoff, callback) {
```

Comment:
`getHistorical` should be implemented using `getOps`?

Reply:
done.